### ISSUE TITLE
Error when launching stack with LoadBalancerController with promtheus+grafana stack

### ISSUE THREAD

**Issue by ThisIsQasim** [aws-quickstart/quickstart-amazon-eks#298]
The namespace resource for prometheus or grafana fails when being launched in a stack that also has AWS Load Balancer Controller stack enabled. The `AWSQS::Kubernetes::Resource` throws the following exception.
```
botocore.errorfactory.ResourceConflictException: An error occurred (ResourceConflictException) when calling the UpdateFunctionCode operation: Conflict due to concurrent requests on this function. Please try this request again.
```

I am guessing the custom resource doesn't support parallel invocations and we need to add a conditional `DependsOn` so that it isn't invoked in parallel.

**Comment by jaymccon** [comment]
I think this is due to a race condition in the resource type which could result in the `AWSQS::Kubernetes::Resource` trying to do an update when another parallel resource has already started an update. I've added an additional check for the error you've quoted. It may take a couple of days to get the update merged into this repo.
